tests/unit/test_container_decoders.py

```diff
@@ -153,3 +238,37 @@ def test_screen_with_plates_decoder(self, screen_with_plates):
         plate_1, plate_2 = v.linkedPlateList()
         self.assert_plate(plate_1, 5L)
         self.assert_plate(plate_2, 6L)
+
+    @pytest.mark.parametrize("color", [(255, 0, 0, 255),
+            (255, 255, 255, 255),
```

Comment:
Is that really a blocker? I was hoping to get this merged ASAP. I can push a commit but then @chris-allan has to pick it etc.